wrangler-api/src/main/java/io/cdap/wrangler/api/RelationalDirective.java

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright © 2017-2019 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+package io.cdap.wrangler.api;
+
+import io.cdap.cdap.etl.api.relational.LinearRelationalTransform;
+import io.cdap.cdap.etl.api.relational.Relation;
+import io.cdap.cdap.etl.api.relational.RelationalTranformContext;
+
+/**
+ * Directive interface which supports Relational transformations
+ */
+public interface RelationalDirective extends Directive, LinearRelationalTransform {
```

Comment:
Did not understand the need for this interface

Reply:
Had pushed this accidentally. It's not needed. Thanks fr pointing it out